contrib/twitchtv/twirp/example_test.go

```diff
@@ -0,0 +1,56 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2019 Datadog, Inc.
+
+package twirp_test
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	twirptrace "gopkg.in/DataDog/dd-trace-go.v1/contrib/twitchtv/twirp"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+
+	"github.com/twitchtv/twirp/example"
+)
+
+func ExampleWrapClient() {
+	// Start the tracer
+	tracer.Start()
+	defer tracer.Stop()
+
+	client := example.NewHaberdasherJSONClient("http://localhost:8080", twirptrace.WrapClient(&http.Client{}))
+	for i := 0; i < 10; i++ {
+		hat, err := client.MakeHat(context.Background(), &example.Size{Inches: 6})
+		if err != nil {
+			fmt.Println("error making hat:", err)
```

Comment:
Can we please add a space after the colon?

Reply:
`fmt.Println` already adds a space between the values.